### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by vineethvarghese**

### CODE REVIEW

**Comment by SamRoberts** [comment]
Did you have to do anything special to get those instructions to work for you?

File: README.md [hunk 1]
```diff
@@ -5,18 +5,18 @@ Scripts used for Continuous Integration
...
 
 1. Add a `.travis.yml` file to the top level directory of the project. See some of the other projects for a sample one.
-1. Download the travis client. https://github.com/travis-ci/travis.rb.
-1. Login. `travis login --pro` or `travis login --org`.
-1. Add the encrypted artifactory password. By running this command in the same directory as `.travis.yml`. `travis encrypt 'ARTIFACTORY_PASSWORD=...'`.
-1. [Optional to publish documentation] Add the encrypted github password. By running this command in the same directory as `.travis.yml`. `travis encrypt 'GH_PASSWORD=...'`.
-1. Enable the build by `travis enable -r [name]`.
+2. Install the travis client. Instructions are available at https://github.com/travis-ci/travis.rb.
+3. Login. `travis login --pro` or `travis login --org`.
+4. From project folder (same directory as `.travis.yml`), enable your project with Travis CI by running the command `travis enable`.
```
Comment (SamRoberts) [inline README.md]: How is the `.travis.yml` created?

Comment (vineethvarghese) [inline README.md]: Well I had to install the client which wasn't really mentioned and then
enable the project before encryption.

Comment (vineethvarghese) [inline README.md]: Well I copied the yml file :)

Comment (SamRoberts) [inline README.md]: Maybe mention to create the yml file, using other projects as a template?

**Comment by SamRoberts** [comment]
:+1:

**Comment by stephanh** [comment]
:+1:

**Comment by stephanh** [comment]
@vineethvarghese do you want to merge this?

**Comment by vineethvarghese** [comment]
Yes I will do it. I am just waiting to get my build sorted on Travis so that I am sure all my steps are correct.

**Pull request merged**